### PR TITLE
Clarify section on Packet I/O messages

### PR DESCRIPTION
**Pull request by smolkaj**
In particular with regards to translated types.

### CODE REVIEW

File: docs/v1/P4Runtime-Spec.mdk [hunk 1]
```diff
@@ -5565,15 +5565,27 @@ message PacketMetadata {
...
-  `ControllerPacketMetadata.metadata` specification. If the `metadata` field
+  `ControllerPacketMetadata.Metadata` message with the same `id`, in the
+  following sense:
+  + If `ControllerPacketMetadata.Metadata.bitwidth` is set, or if
+    `ControllerPacketMetadata.Metadata.type_name` is set and
+    `P4NewTypeSpec.translated_type.sdn_bitwidth` is set in the `P4NewTypeSpec`
+    for the type name, then `PacketMetadata.value` must be a binary string of
+    the specified bit width conforming to the [Bytestrings](#sec-bytestrings)
+    requirements.
+  + If `ControllerPacketMetadata.Metadata.type_name` is set and
```
Comment (chrispsommers) [inline docs/v1/P4Runtime-Spec.mdk:5583]: Again, a worked-out example might help here.

**Review by chrispsommers — APPROVED** [review]
LGTM. Suggest to add some more examples to illustrate some cases involving `P4NewTypeSpec`

**Comment by smolkaj** [comment]
I agree that adding examples would be useful. 
Unfortunately I'm a bit pressed on time right now. 
We could submit this imperfect PR now (assuming we agree that it is an incremental improvement), or I would be happy to add examples once I'm back from vacation.

**Comment by chrispsommers** [comment]
Sure, that's why I approved, you can put this idea in backlog. Feel free to merge.

**Comment by smolkaj** [comment]
Sounds good. Thanks.

**Pull request merged**